### ISSUE TITLE
TODO: Add title

### ISSUE THREAD

**Issue by jiashen0** [koalaman/shellcheck#1153]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/your/shebang
Your script
```

#### Here's what shellcheck currently says:
```
TODO: Describe current output
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output
